docs/docker_tips.md

```diff
@@ -70,10 +70,19 @@ or
 $ docker-compose up -d
 ```
 
-### Deleting Docker images
+### Deleting Docker containers/images/volumes
```

Comment:
might want to add a BURN IT WITH FIRE option for when you just want docker to get rid of everything
`docker system prune -af; docker volume prune -f`